spec/std/spec/cli_spec.cr

```diff
@@ -0,0 +1,183 @@
+require "spec"
+require "spec/cli"
+
+private def prepare_cli(argv, **kargs)
+  cli = Spec::CLI.new(argv, **kargs)
+  cli.prepare
+
+  {cli, cli.options}
+end
+
+# override and capture exit calls
+module Spec
+  class CLI
```

Comment:
This could be a subclass of `Spec::CLI` which is only used for testing and has appropriate overwrites.

Reply:
Ah, didn't thought that way. Care to suggest a name for such class?

Thank you.

Reply:
`class TestCLI < Spec::CLI`?